instrumentation/opentelemetry_phoenix/mix.exs

```diff
@@ -0,0 +1,68 @@
+defmodule OpentelemetryPhoenix.MixProject do
+  use Mix.Project
+
+  def project do
+    [
+      app: :opentelemetry_phoenix,
+      description: description(),
+      version: "1.0.0-rc.2",
+      elixir: "~> 1.10",
+      start_permanent: Mix.env() == :prod,
+      dialyzer: [
+        plt_add_apps: [:ex_unit, :mix],
+        plt_core_path: "priv/plts",
+        plt_local_path: "priv/plts"
+      ],
+      deps: deps(),
+      name: "Opentelemetry Phoenix",
+      docs: [
+        main: "OpentelemetryPhoenix",
+        extras: ["README.md"]
+      ],
+      elixirc_paths: elixirc_paths(Mix.env()),
+      package: package(),
+      source_url: "https://github.com/opentelemetry-beam/opentelemetry_phoenix"
+    ]
+  end
+
+  # Run "mix help compile.app" to learn about applications.
+  def application do
+    [
+      extra_applications: []
+    ]
+  end
+
+  defp description do
+    "Trace Phoenix requests with OpenTelemetry."
+  end
+
+  defp package do
+    [
+      description: "OpenTelemetry tracing for the Phoenix Framework",
+      files: ~w(lib .formatter.exs mix.exs README* LICENSE* CHANGELOG*),
+      licenses: ["Apache-2.0"],
+      links: %{
+        "GitHub" => "https://github.com/opentelemetry-beam/opentelemetry_phoenix",
+        "OpenTelemetry Erlang" => "https://github.com/open-telemetry/opentelemetry-erlang",
+        "OpenTelemetry.io" => "https://opentelemetry.io"
+      }
+    ]
+  end
+
+  defp elixirc_paths(:test), do: ["lib", "test/support"]
+  defp elixirc_paths(_), do: ["lib"]
+
+  # Run "mix help deps" to learn about dependencies.
+  defp deps do
+    [
+      {:opentelemetry_api, "~> 1.0.0-rc"},
+      {:opentelemetry, "~> 1.0.0-rc"},
```

Comment:
Shouldn't this be only `dev/test` dependency? I'd assume only opentelemetry_api is needed for all the calls, and the user could use whichever implementation they choose (though I guess there's only one atm).